src/Bundle/Form/DataTransformer/ResourceToIdentifierCacheableTransformer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Form\DataTransformer;
+
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Webmozart\Assert\Assert;
+
+final class ResourceToIdentifierCacheableTransformer implements DataTransformerInterface
+{
+    private RepositoryInterface $repository;
+
+    private string $identifier;
+
+    /** @var array<ResourceInterface> */
+    private static array $cache;
```

Comment:
```suggestion
    private array $cache;
```
Pretty sure making it static could potentially leak objects between different instances.
To leave it as static you'd need to prefix the key w/ a class name, make it a nested array or something like that.

Reply:
Is it possible or needed to make use of psr CacheInterface/symfony cache for this?